test/system/030-run.bats

```diff
@@ -736,4 +736,16 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+
+@test "podman run with FIPS enabled" {
+    skip_if_rootless
+    skip_if_fips_disabled
+    ELASTIC_IMAGE="docker.elastic.co/elasticsearch/elasticsearch-oss:7.9.2-amd64"
```

Comment:
I have no idea about this, because I can't reproduce related bug for alpine and registry:2 image, and I also can't reproduce it w/ elasticsearch image, so I will use quay.io/libpod/registry:2 image instead.